nodeJS/orms/prisma_orm.md

```diff
@@ -0,0 +1,157 @@
+
+### Prisma Schema
+
+The Prisma schema is a file where you will define your models. For example, consider a message table in a chat app:
+
+```text
+model Message {
+   id        Int      @id @default(autoincrement())
+   content   String   @db.VarChar(255) 
+   createdAt DateTime @default(now())
+   author    User     @relation(fields: [authorId], references: [id])
+   authorId  Int     
+}
+
+model User {
+   // user's fields
+}
+```
+
+There's a bunch of new stuff up here. Prisma schema files are written in Prisma Schema Language. You can see how not only are there column definitions for the table, but also a relation to another table defined within the `Message` model. You'll learn more about this in the assignment readings.
+
+This schema file lives in your codebase and is tracked by version control. You can guess by now how useful this is.
+
+### Prisma Client
+
+The client is a separate library that you will use to interact with your database. The Prisma client is a bit special in that it's customized to your schema. Um... what?
+
+```javascript
+// instantiate the client
+import { PrismaClient } from '@prisma/client';
+const prisma = new PrismaClient();
+
+// when creating a new message
+await prisma.message.create({
+   data: {
+      content: 'Hello, world!',
+      authorId: 1
+   }
+})
+
+// when fetching all messages
+const messages = await prisma.message.findMany();
+```
+
+Notice the `prisma.message` object? How did Prisma Client know that there's a `message` model? Once you create or update the schema file, all you have to do is run
+
+```bash
+npx prisma generate
+```
+
+in the CLI, and Prisma ORM will generate the client for you. The client can handle all sorts of querying: joins, filters, sorting, pagination, and more.
+
+You might have a complex query that you just are unable to get right via the Prisma client, or if you feel more comfortable writing a query raw, you can do that too. Prisma Client supports raw queries as well.
+
+### Prisma Migrate
+
+Prisma migrate is a tool that helps you perform database migrations. You won't be using it a whole ton in the curriculum, but it's good to be aware of it. When you decide to change the schema in any way, you run a Prisma migration to apply the schema changes to the database. These changes are tracked in a `migrations` folder in your codebase.
+
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+1. Go through the [get started guide for relational databases](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-postgresql). It covers migrations, schema, and the Prisma client. Make sure you select "JavaScript" as the language and "PostgreSQL" as the database.
```

Comment:
Changing the href should have it land on JS and Postgresql automatically, instead of switching from TS.

```suggestion
1. Go through the [get started guide for relational databases](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases-node-postgresql), which covers migrations, schema, and the Prisma client.
```